pretyping/vnorm.ml

```diff
@@ -313,8 +310,8 @@ and nf_predicate env sigma ind mip params v pT =
       let params = if Int.equal n 0 then params else Array.map (lift n) params in
       let dom = mkApp(mkIndU ind,Array.append params rargs) in
       let body = nf_vtype (push_rel (LocalAssum (name,dom)) env) sigma vb in
-      true, mkLambda(name,dom,body)
-    | _ -> false, nf_val env sigma v crazy_type
+      mkLambda(name,dom,body)
+    | _ -> nf_val env sigma v crazy_type
```

Comment:
Thinking about it, shouldn't this pattern be an `assert false`? The return predicate is supposed to always be dependent now so there shouldn't be a shortage of lambdas.

Reply:
Come on, you fixed the linting but didn't set the assertion failure?

Reply:
I can look at the code but not now. Fixing the whitespace didn't require opening the file.